app/models/verification/management/email.rb

```diff
@@ -21,7 +21,7 @@ def user?
   def save
     return false unless valid?
 
-    plain_token, encrypted_token = Devise.token_generator.generate(User, :email_verification_token)
+    encrypted_token, plain_token = Devise.token_generator.generate(User, :email_verification_token)
```

Comment:
Same here; instead of changing the variable names, we could use `email_verification_token: encrypted_token` in the `user.update!` call.